plugins/BEdita/API/src/Controller/AppController.php

```diff
@@ -75,6 +76,9 @@ public function initialize()
             'storage' => 'Memory',
         ]);
         $this->Auth->allow();
+        if ($this->name !== 'Login') {
+            LoggedUser::setUser($this->Auth->identify());
```

Comment:
After #968, when we will be able to remove `$this->Auth->allow()` just above these lines, this could be probably bound to `Auth.afterIdentify` event.